cmd/kyma/test/run/cmd.go

```diff
@@ -45,13 +53,21 @@ To execute all test defintions, run ` + "`kyma test run -n example-test`" + `.
 	cobraCmd.Flags().Int64VarP(&o.ExecutionCount, "count", "c", 1, `Defines how many times every test should be executed. "count" and "max-retries" flags are mutually exclusive.`)
 	cobraCmd.Flags().Int64VarP(&o.MaxRetries, "max-retries", "", 0, `Defines how many times a given test is retried when it fails. A suite is marked with a "succeeded" status even if some tests failed at first and then finally succeeded. The default value of 0 means that there are no retries of a given test.`)
 	cobraCmd.Flags().Int64VarP(&o.Concurrency, "concurrency", "", 1, "Specifies the number of tests to be executed in parallel.")
+	cobraCmd.Flags().DurationVar(&o.Timeout, "timeout", 0, `Maximum time during which the test suite is being watched, zero means infinite. Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".`)
+	cobraCmd.Flags().BoolVarP(&o.Watch, "watch", "w", o.Watch, "Watch the status of the test suite until it finishes or the defined `--timeout` occurs.")
```

Comment:
Shouldn't here the default value be `false`?

Reply:
It depends. In kubectl they always use the zero-value of the given option,  so in this case `o.Watch` but I've check your approach and you always set the "specified" defaults. To be consistent I adjusted that to be `false`

Reply:
Yes, I know about that in kubectl. But we decided to put explicit values to make it simpler and more readable.